webrender/src/render_backend.rs

```diff
@@ -347,6 +347,14 @@ impl RenderBackend {
                                 frame_counter += 1;
                             }
                         }
+                        ApiMsg::ExternalEvent(evt) => {
+                            let mut notifier = self.notifier.lock();
+                            notifier.as_mut()
```

Comment:
this first `as_mut` is not needed either here or in any other mentions of `notifier`